src/lib/getData.ts

```diff
@@ -144,8 +155,17 @@ export async function getCourseContributors(
   const { DataStore } = withSSRContext(context);
 
   const contributorCourses = await DataStore.query(ContributorCourse);
+  const resolvedCourses = [];
+
+  for (const contributorCourse of contributorCourses) {
+    const id = await contributorCourse.id;
+    const contributor = await contributorCourse.contributor;
+
+    const course = await contributorCourse.course;
+    resolvedCourses.push({id, contributor:contributor, course: course}); 
+  }
```

Comment:
I'm a little confused by this loop, why do we have it? It looks like it's just accessing the properties from `contributorCourses` and just making a new object array to filter? Is there a reason we need to `await` the properties too?